### ISSUE TITLE
Download hg38/ucsc

### ISSUE THREAD

**Issue by k-tipp** [charite/jannovar#415]
**Executable**:
jannovar-cli version 0.26 from [here](https://search.maven.org/search?q=jannovar) 

**Commands:**
`java -jar -Xmx16000m -Xms3000m -d64 jannovar-cli-0.26.jar download -d hg38/ucsc`
and
`java -jar -Xmx16000m -Xms3000m -d64 jannovar-cli-0.26.jar download -d hg38/ensembl`

**Exception:**
```
Exception in thread "main" java.lang.IllegalStateException: Not connected
        at sun.net.ftp.impl.FtpClient.issueCommand(Unknown Source)
        at sun.net.ftp.impl.FtpClient.issueCommandCheck(Unknown Source)
        at sun.net.ftp.impl.FtpClient.setType(Unknown Source)
        at sun.net.ftp.FtpClient.setBinaryType(Unknown Source)
        at sun.net.www.protocol.ftp.FtpURLConnection.getInputStream(Unknown Source)
        at de.charite.compbio.jannovar.datasource.FileDownloader.copyURLToFileThroughURL(FileDownloader.java:230)
        at de.charite.compbio.jannovar.datasource.FileDownloader.copyURLToFile(FileDownloader.java:83)
        at de.charite.compbio.jannovar.datasource.JannovarDataFactory.build(JannovarDataFactory.java:87)
        at de.charite.compbio.jannovar.cmd.download.DownloadCommand.run(DownloadCommand.java:44)
        at de.charite.compbio.jannovar.Jannovar.main(Jannovar.java:64)

```
**Reason:**
The file ftp://ftp.ncbi.nlm.nih.gov/genomes/H_sapiens/Assembled_chromosomes/chr_accessions_GRCh38.p7 doesn't exists anymore (see: ftp://ftp.ncbi.nlm.nih.gov/genomes/H_sapiens/Assembled_chromosomes/).

**Comment by visze** [comment]
Hi k-tipp,

thank you for finding that issue and sorry for the really late answer. We forgot to update the genome build from p7 to p12 for ucsc and ensembl

You can download it by using your own `.ini` file with the corrected paths. Please download this file:
https://github.com/charite/jannovar/blob/develop/jannovar-cli/src/main/resources/default_sources.ini
and change all text with 
`ftp://ftp.ncbi.nlm.nih.gov/genomes/H_sapiens/Assembled_chromosomes/chr_accessions_GRCh38.p7` 
to
`ftp://ftp.ncbi.nlm.nih.gov/genomes/H_sapiens/Assembled_chromosomes/chr_accessions_GRCh38.p12`.

Then use the following command to download the database:

```bash
java -jar -Xmx16000m -Xms3000m -d64 jannovar-cli-0.26.jar download -d hg38/ucsc -s path/to/your/edited/ini/default_sources.ini
```
and
```bash
java -jar -Xmx16000m -Xms3000m -d64 jannovar-cli-0.26.jar download -d hg38/ensemb -s path/to/your/edited/ini/default_sources.ini
```

**Comment by holtgrewe** [comment]
Fixed in 96e57b9f / develop.
